### DISCUSSION TITLE
NXdata with auxiliary_signals saying Expected dataset to have numeric type - myHDF5

### DISCUSSION THREAD

**Discussion by joaonizer** [silx-kit/h5web#1669] [Q&A] [answered]
Hi,

I have a group `SParameters` that is a `NXentry` and subgroups `S11`,..,`SNN` that are `NXdata`. Each one of them has their own `data`. Within `SParameters` I've created `SoftLink()`'s to the `data` in the other subgroups so I can plot them all using `auxiliary_signals` in a new `NXdata` called `ALL`. You can see them all here:

![image](https://github.com/silx-kit/h5web/assets/26409849/37f8341c-f6b4-4bf7-91ef-3cb83cf3b61e)

Unfortunately, the  display says "Expected dataset to have numeric type"

I am following the example under `spectrum_with_aux` in [h5web:mock](https://h5web.panosc.eu/mock):

![image](https://github.com/silx-kit/h5web/assets/26409849/e8d7e176-3d38-44a6-8635-2f9849a3ff25)

![image](https://github.com/silx-kit/h5web/assets/26409849/d8c6ad46-add2-4585-8ee9-56e3e398b3b1)

`axes` is not an array as the data is 1D. I even tried adding `"."` and making it an array but it did not work.

Here is the .hdf5 file:
[data_file.zip](https://github.com/user-attachments/files/15809334/wg_straight_2024-06-12.18-11-20-493584_e5a788a0-cb5e-4141-81ab-e418940d8feb.zip)

Again, thanks for the help! 🙇‍♂️

**Comment by axelboc** [comment, marked as answer]
Hmm, I think you've hit an unsupported feature: **complex auxiliary signals**. I'll turn this discussion into an issue. Thanks for the detailed report 👍

**Reply by joaonizer** [reply]
😄 Thanks @axelboc! I though `auxiliary_signals` would be handled the same as the `signal` itself. Anyways, looking forward for this feature.

**Reply by loichuder** [reply]
Ouch, nice edgecase. Thanks for the report.

**Reply by axelboc** [reply]
Complex auxiliary signals are now supported in [myHDF5](https://myhdf5.hdfgroup.org/). Please let us know if you encounter any issues.
